conf/pf.conf.defaults

```diff
@@ -1364,7 +1364,10 @@ fingerbank_data_update_interval=6h
 #
 # Interval at which PacketFence updates the local Fingerbank data from the cloud API
 cluster_check_interval=1m
-
+# maintenance.option_82_task_interval
+#
+# Interval at which PacketFence will query compatibles switches for option 82 switch mac address
+option_82_task_interval=6D
```

Comment:
We have to reduce this value to something like 23h since the switch cache is set to 1D